<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="BA-000235932" name="com.telus.wnp.tests.FixReviewComments" tests="1" failures="1" timestamp="2022-05-04T15:51:27 IST" time="61.314" errors="0">
  <testcase name="testMethod_169244_PortInModifyFromSOA" time="61.314" classname="com.telus.wnp.tests.FixReviewComments">
    <failure type="java.lang.AssertionError" message="java.lang.AssertionError: Exception Occured: Unable to verify subscriber status from CODA expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: java.lang.AssertionError: Exception Occured: Unable to verify subscriber status from CODA expected [true] but found [false]
at com.test.ui.actions.Validate.assertTrue(Validate.java:1911)
at com.test.ui.actions.Validate.assertTrue(Validate.java:1924)
at com.telus.wnp.steps.CODASearchPageSteps.verifySubscriberStatusFromCODA(CODASearchPageSteps.java:113)
at com.telus.wnp.tests.FixReviewComments.testMethod_169244_PortInModifyFromSOA(FixReviewComments.java:289)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.AssertionError: Exception Occured: Unable to verify subscriber status from CODA expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at com.test.ui.actions.Validate.assertTrue(Validate.java:1907)
... 31 more
]]>
    </failure>
  </testcase> <!-- testMethod_169244_PortInModifyFromSOA -->
  <system-out/>
</testsuite> <!-- com.telus.wnp.tests.FixReviewComments -->
